templates/flavors/clusterclass-rke2/azure-machine-template-controlplane.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: AzureMachineTemplate
+metadata:
+  name: ${CLUSTER_NAME}-control-plane
+spec:
+  template:
+    spec:
+      image:
+        computeGallery:
+          gallery: replace-me
+          name: replace-me
+          version: replace-me
```

Comment:
Was this left intentionally with `replace-me` value ?

Reply:
This was replaced by CC patch, but it turns out that having this field set by default causes errors on double template apply, which might have affected e2e execution. Replaced with full template patch now.